modules/cnv_upload_virtctl.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[upload-vmdisk-virtctl]]
+= Uploading a local disk image to a new PVC
+
+You can use `virtctl image-upload` to upload a virtual machine disk image from
+a client machine to your {product-title} cluster. This creates a PVC that can be
+associated with a virtual machine after the upload has completed.
+
+.Prerequisites
+
+* A virtual machine disk image, in RAW or QCOW2 format. It can be compressed
+using *xz* or *gzip*.
+* *kubevirt-virtctl* must be installed on the client machine.
+* The client machine must be xref:../install_config/router/default_haproxy_router.html#overview[configured]
+to trust the OpenShift router's certificate.
+
+.Procedure
+
+. Identify the following items:
+* File location of the VM disk image you want to upload
+* Name and size desired for the resulting PVC
+
+. Use the `virtctl image-upload` command to upload your VM image,
+making sure to include your chosen parameters. For example:
++
+----
+$ virtctl image-upload --uploadproxy-url=https://$(oc get route cdi-uploadproxy-route -n cdi -o=jsonpath='{.status.ingress[0].host}') --pvc-name=upload-fedora-pvc --pvc-size=10Gi --image-path=/images/fedora28.qcow2
+----
++
+[CAUTION]
+====
+To allow insecure server connections when using HTTPS, use the `--insecure`
+parameter. Be aware that when you use the `--insecure` flag, the authenticity of
+the upload endpoint is *not* verified.
+====
+
```

Comment:
Maybe mention additional/optional options? Like `--access-mode`... or maybe `-h` to get help?

Reply:
I don't want to overly broaden the scope of this PR.  --help is a standard argument so I think we can expect users to know it.  As for documenting --access-mode and others, we would need to open a new PR and this late in the game it will probably need to wait.